packages/workbox-recipes/src/warmStrategyCache.ts

```diff
@@ -0,0 +1,30 @@
+import { Strategy } from 'workbox-strategies/src/Strategy';
+
+import './_version.js';
+
+export interface WarmStrategyCacheOptions {
+  paths: Array<string>;
```

Comment:
Can you switch from calling this `paths` to `urls`?

(As far as I can tell, you should be able to pass in any absolute or relative URL inside this array, right?)